lib/sync/config.go

```diff
@@ -15,6 +15,7 @@ const (
 	FetchTimeout                    = 1 * time.Minute
 	RetryInterval                   = 10 * time.Second
 	CheckBlockHeightInterval        = 30 * time.Second
+	CheckPrevBlockInterval          = 10 * time.Minute
```

Comment:
@anarcher You think 10 minutes is best? I did not think deeply yet, but 10 minutes is too long. How about 30 seconds?